inspectit-oce-documentation/src/docs/asciidoc/_includes/appendix/index.adoc

```diff
@@ -0,0 +1,8 @@
+== Appendix
+
+=== All Configuration Options
+
+[source,yaml]
+----
+include::../{resource-dir}/default.yaml[]
```

Comment:
This just prints 
"../../../../inspectit-oce-core/src/main/resources/config/default.yml"
in the result html for me. I guess this is not intended?